lib/runtime/ops.js

```diff
@@ -247,6 +275,8 @@ var ops = {
             return JuttleMoment.lte(left, right);
         } else if (values.isDuration(left) && values.isDuration(right)) {
             return JuttleMoment.lte(left, right);
+        } else if (values.isArray(left) && values.isArray(right)) {
+            return values.equal(left, right) || this.lt(left, right);
```

Comment:
I don’t like the asymmetry between the definitions of `lt`/`gt` and `lte`/`gte`. At the same time I don’t like the big chunk of code in `lt`/`gt` (it feels like something which should be in a separate function) and the duplication of code in all the implementation of comparison operators.

What about implementing the comparison as a separate function in `values.js`, named `values.compare`? It would return the usual `strcmp`-like result and the operators would than be implemented like this:

```javascript
lt: function(left, right) {
    return values.compare(left, right) < 0;
}
```

This solution has some performance overhead, but I don’t think it will matter much in practice and we can always inline `values.compare` and revert to duplicate code if needed.